### PR TITLE
feat: add requirements

### PR DESCRIPTION
**Pull request by davidbp**
Add tfidf text encoder

### CODE REVIEW

File: tests/unit/test_tfidf.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from jina import Executor
+
+def test_tfidf():
+    encoder = Executor.load_config('../../config.yml')
```
Comment (JoanFM) [inline tests/unit/test_tfidf.py]: please use `cur_dir`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/unit/test_tfidf.py (hunk 1 above)
Comment (davidbp) [reply tests/unit/test_tfidf.py]: good catch, updated

File: tfidf_text_executor.py [hunk 2]
```diff
@@ -28,9 +43,39 @@ def __init__(
             )
 
     @requests
-    def encode(self,docs: DocumentArray,  *args, **kwargs) -> DocumentArray:
-        iterable_of_texts = docs.get_attributes('text')
-        embedding_matrix = self.tfidf_vectorizer.transform(iterable_of_texts)
+    def encode(self, docs: Optional[DocumentArray],  *args, **kwargs):
```
Comment (JoanFM) [inline tfidf_text_executor.py]: what about `parameters` to override `traversal_path`

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tfidf_text_executor.py (hunk 2 above)
Comment (davidbp) [reply tfidf_text_executor.py]: good catch, updated

File: tests/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+git+https://github.com/jina-ai/jina.git@v2.0.0rc6#egg=jina[http]
```
Comment (JoanFM) [inline tests/requirements.txt]: I think jina is not needed here, as it is installed in the no tests requirements

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/requirements.txt (hunk 3 above)
Comment (davidbp) [reply tests/requirements.txt]: true, just removed it

**Review by JoanFM — APPROVED** [review]

**Pull request merged**